src/blueprints/tv2/value-objects/tv2-action.ts

```diff
@@ -48,6 +49,13 @@ export interface Tv2CameraAction extends Tv2PartAction {
   }
 }
 
+export interface Tv2RemoteAction extends Tv2PartAction {
+  metadata: {
+    contentType: Tv2ActionContentType.REMOTE,
+    remoteNumber: string
```

Comment:
I assume this is for something in the frontend?

Reply:
Currently it is what is used to determine what key binding to assign the remote action to. When we get a customizable key binding framework, this can be removed.